test/XRP/xrp-protocol-buffer-conversion.test.ts

```diff
@@ -478,14 +506,17 @@ describe('Protocol Buffer Conversion', function (): void {
 
   it('Convert PAYMENT Transaction with bad payment fields', function (): void {
     // GIVEN a GetTransactionResponse protocol buffer with Transaction payment fields which are incorrect
-    // WHEN the protocol buffer is converted to a native TypeScript type.
-    const transaction = XrpTransaction.from(
-      testInvalidGetTransactionResponseProto,
-      XrplNetwork.Test,
+    // WHEN the protocol buffer is converted to a native TypeScript type THEN an error is thrown.
+    assert.throws(
+      () => {
+        XrpTransaction.from(
+          testInvalidGetTransactionResponseProto,
+          XrplNetwork.Test,
+        )
+      },
+      XrpError,
+      'Cannot construct BigInt from IssuedCurrency protobuf `value` field.',
```

Comment:
can we remove the rest of the error message checks as well?

Reply:
Oops missed this one